db/seeds.rb

```diff
@@ -75,7 +75,50 @@
   visible_image: true
   ))
 
-# Income starts at 6
+
+# Insert Team Roles
+TeamRole.create(
```

Comment:
I'm not a fan of storing game logic in seed data.  The role_permissions and display_name are generic and won't change much from Game to Game.  As such we should store them as a hash/constant in the Users.rb and then reference them in the seed.  Eventually, we would move them into a yml or json file (although that would be far into the future).